include/fuse.h

```diff
@@ -293,23 +296,28 @@ struct fuse_config {
 	 *  enabling this setting, all direct writes on the same file are
 	 *  serialized, resulting in huge data bandwidth loss).
 	 */
-	int parallel_direct_writes;
+	uint32_t parallel_direct_writes;
+
+	/**
+	 * The timeout in seconds for which file attributes are cached
+	 * for the purpose of checking if auto_cache should flush the
+	 * file data on open.
+	 */
+	int32_t ac_attr_timeout_set;
```

Comment:
If changing the order of these two `ac_attr_*` members was intended, then I think that should be clearly documented by doing this in a separate commit. Currently, this is done in the commit labeled `Convert to uint32_t/int32_t in struct fuse_config`.

Reply:
Thanks, as explained below, I updated the commit message.